src/test/suite/participant/participant.test.ts

```diff
@@ -1507,6 +1560,152 @@ suite('Participant Controller Test Suite', function () {
               'see previous messages'
             );
           });
+
+          suite('with an empty database name', function () {
```

Comment:
These tests seem very similar to the ones we have for `/query` - can we parameterize them somehow rather than copy-paste?